_data/changelogs/tooltip.yml

```diff
@@ -19,4 +19,31 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.2.0 release
+  - date: 2022-10-19
+    summary: Tooltip allows for dynamic initialization.
+    summaryAdditional: Tooltip initializes on first interaction and checks if component has been initialized before toggling. If it hasn't initialized properly it will call a setup attributes function.
+    affectsAccessibility: false
+    affectsAssets: false
+    affectsGuidance: false
+    affectsJavascript: true
+    affectsMarkup: false
+    affectsStyles: true
```

Comment:
Just confirming `affectsStyles` here. It seems to me to be a JS update only.

Reply:
Correct, should be `false`. Must've had it confused with the other JS work around the same time!